deep_qa/data/dataset_readers/squad_sentence_selection_reader.py

```diff
@@ -20,26 +22,107 @@ def main():
                                                   "use by the SentenceSelectionInstance"))
     parser.add_argument('input_filename', help='Input SQuAD json file.')
     parser.add_argument('--output_directory',
-                        help=('Output directory. Make sure to end the string with a /'))
+                        help='Output directory. Make sure to end the string with a /')
+    parser.add_argument('--negatives', default='paragraph', help="See class docstring")
     args = parser.parse_args()
-    reader = SquadSentenceSelectionReader(args.output_directory)
+    reader = SquadSentenceSelectionReader(args.output_directory, args.negatives)
     reader.read_file(args.input_filename)
 
 
 class SquadSentenceSelectionReader():
-    def __init__(self, output_directory=None):
+    """
+    Parameters
+    ----------
+    output_directory: str, optional (default=None)
+        If you want the output stored somewhere other than in a ``processed/`` subdirectory next to
+        the input, you can override the default with this parameter.
+    negative_sentence_selection: str, optional (default="paragraph")
+        A comma-separated list of methods to use to generate negative sentences in the data.
+
+        There are three options here:
+
+        (1) "paragraph", which means to use as negative sentences all other sentences in the same
```

Comment:
> And in general, more options to play around with stuff doesn't hurt anything, so why assume that "paragraph" is always done, if it's not already written that way?

sure, I personally feel like more options make things harder to break / use :) but I understand the rationale behind these options and I'm fine with it.